### PR TITLE
Add getChainTopUserTransactions.graphql

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

Add the query getChainTopUserTransactions and e2e test

### Test Plan

pnpm test
pnpm fmt

### CODE REVIEW

**Comment by 0xjinn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #35** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/35" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/35?utm_source=stack-comment).

**Review by gregnazario — COMMENTED** [review]
I'm wondering how important this is to be in the SDK?

I'm assuming this is only useful for explorers and statistic sites

**Comment by 0xjinn** [comment]
> I'm wondering how important this is to be in the SDK?
> 
> I'm assuming this is only useful for explorers and statistic sites

@gregnazario Good question. Yeah I don't think this is that much of a P0 query. Shall we revisit the priority on these queries? - https://www.notion.so/aptoslabs/fetch-queries-to-port-over-into-sdk-v2-8a38cf2c47e542e2ae299c900894277b

File: src/types/generated/operations.ts [hunk 1]
```diff
@@ -1,336 +1,93 @@
-import * as Types from "./types";
+import * as Types from './types';
 
-export type CurrentTokenOwnershipFieldsFragment = {
-  token_standard: string;
-  token_properties_mutated_v1?: any | null;
-  token_data_id: string;
-  table_type_v1?: string | null;
```
Comment (gregnazario) [inline src/types/generated/operations.ts:7]: Any reason this file just disappears?

File: tests/e2e/api/general.test.ts [hunk 2]
```diff
@@ -103,4 +103,12 @@ describe("general api", () => {
 
     expect(chainId.ledger_infos[0].chain_id).toBe(2);
   });
+
+  test("it should fetch chain top user transactions", async () => {
+    const config = new AptosConfig({ network: Network.TESTNET });
```
Comment (gregnazario) [inline tests/e2e/api/general.test.ts:108]: This should also be local

**Review by gregnazario — APPROVED** [review]
Let's please get these PRs in, and we may think about which queries we need in the future on here.

File: src/types/generated/operations.ts (hunk 1 above)
Comment (0xjinn) [reply src/types/generated/operations.ts:7]: hmm wait, which file? or you mean the `CurrentTokenOwnershipFieldsFragment` type? I think it got re-formatted

**Pull request merged**